OpenRA.Mods.Common/Terrain/DefaultTileCache.cs

```diff
@@ -126,13 +126,13 @@ public Theater(TileSet tileset, Action<uint, string> onMissingImage = null)
 				var allSprites = variants.SelectMany(s => s);
 
 				// Ignore the offsets baked into R8 sprites
```

Comment:
Is this something that could be moved to mod code already?

Reply:
Yes, but like I said in the second paragraph of the description I wanted to limit this already large PR to just wrapping/moving the existing code and leave the next set of reworks to followup PRs to keep this one rebaseable and reviewable.